src/components/molecules/NavSearchBar/NavSearchBar.scss

```diff
@@ -1,8 +1,19 @@
 @import "scss/constants.scss";
 
+$search-results-width: 18.125rem;
+
 .nav-search-links {
-  display: flex;
   position: relative;
+  width: $search-results-width;
+
+  .nav-search-input {
+    z-index: 2;
+  }
+
+  .input-field__input {
+    text-align: left;
+    font-size: 0.9em;
+  }
```

Comment:
How does this differ from `nav-search-input` above?

Reply:
`nav-search-input` is the class name we use as the prop for `InputField`, and for <input /> itself we don't have any classNames specified except its `input-field__input`. I think it'd make more sense to add one more prop to specify this className as well, right? 

```html
<!-- InputField -->
<div ref={ref} className={wrapperClassNames}>
      <input className="input-field__input" {...extraInputProps} />
...icons
</div>
```

Reply:
Ah.. I think I see what you're saying here. Basically we can only currently supply custom wrapper classNames to `InputField`, whereas we want to also be able to pass custom classes to the input field itself?

- https://github.com/sparkletown/sparkle/blob/staging/src/components/atoms/InputField/InputField.tsx#L42-L49

If so, then yeah, this is an enhancement we should make to `InputField`. We should rename the current `className` prop to `containerClassName` or similar, and then add a new `inputClassName` or similar. I think there should be some examples of this pattern in the codebase already, so would be good to see what naming pattern we used there, and keep it consistent here as well.

Reply:
There is ImageInput in the project with a similar situation, the class names there: `containerClassName` and `imageClassName`, so in my component I used `containerClassName` and `inputClassName`
see: https://github.com/sparkletown/sparkle/pull/1328/commits/78b78217d1dc217f5d5cd9dd854c92b4d4f34da7

<hr />
but I wonder if we can use just `className` instead of `inputClassName`.

if we look at the component, we can see that in addition to `iconStart` / `iconEnd` props we can use any input-prop (see `extraInputProps`): and it can include `className` as well. 
```html
<input className={inputClassNames} {...extraInputProps} />
```

And one more thing that it'd be just more comfortable (or more familiar)  to use `className` instead of searching for a correct prop name (`inputClassName`).

Reply:
> but I wonder if we can use just `className` instead of `inputClassName`.

While we could definitely use the 'spread' of `extraInputProps` to add the `className`, it's probably a better 'API' to stick with `inputClassName` as you chose. That way it makes it really obvious when using the component which prop ends up where, without having to dive into the code to look.

So my preference would be to keep it as you have it now, using `containerClassName` + `inputClassName`